librosa/util/files.py

```diff
@@ -238,24 +238,24 @@ def find_files(
         ext = [ext]
 
     # Cast into a set
-    ext = set(ext)
+    ext_set = set(ext)
```

Comment:
With the `allow_redefinition` flag, it shouldn't be necessary to use a different name, right?

Using a different name is slightly error prone, because one could accidentally write `e in ext` somewhere below, resulting in a slow O(N) check by accident. In other modern languages (e.g. Rust) it is even encouraged to use symbol shadowing to avoid such mistakes, and in my opinion `allow_redefinition` is also closer to idiomatic Python.

Unfortunately pyright doesn't support that, but only mypy really matters.

Reply:
One would think so, but we have allow_redefinition set in the setup.cfg and it still complains.

Reply:
I've tried this on mypy playground and the error disappears when enabling the option:

![image](https://user-images.githubusercontent.com/3620703/208407082-10b2cbd9-d271-442d-88c9-250b9f0ecb31.png)

I think the setting is actually not active here because the `mypy.ini` is in the tests subfolder and not the root directory. I would move it into the root directory so that in can affect all code consistently.

Reply:
> Why not merge them together and move them to the top-level?

This is the plan once everything works.  For now, I'm keeping them separate.



> Perhaps mypy still doesn't really pick up that setting?

It ought to be - the other settings are being picked up anyway.

Reply:
Ok, got this figured out - turned out to be an environment issue on my side.